xilem_web/src/after_update.rs

```diff
@@ -59,29 +111,67 @@ where
         ctx: &mut ViewCtx,
         element: Mut<'el, Self::Element>,
     ) -> Mut<'el, Self::Element> {
-        let id = ViewId::new(0); // FIXME: what's the right number here?
-        ctx.with_id(id, |ctx| {
-            let rebuild_outcome =
-                self.element
-                    .rebuild(&prev.element, &mut view_state.child_state, ctx, element);
-
-            // FIXME:
-            // if *element != view_state.element {
-            //     view_state.element = element.clone();
-            // }
-
-            let view_path = ctx.view_path().to_vec();
-            ctx.after_update
-                .entry(id)
-                .and_modify(|e| e.0 = true)
-                .or_insert((true, view_path));
-
-            rebuild_outcome
-        })
+        let element = self
+            .element
+            .rebuild(&prev.element, view_state, ctx, element);
+        (self.callback)(element.node);
```

Comment:
But we can do that at least here without breaking stuff (needs the trait `DomNode` in scope):
```suggestion
        element.node.apply_props(element.props);
        (self.callback)(element.node);
```